mxgo/schemas.py

```diff
@@ -479,9 +480,76 @@ class ReplyCandidate(BaseModel):
     rationale: str = Field(description="Brief rationale for the response (≤12 words)")
 
 
+class NewsletterUsageInfo(BaseModel):
+    """Usage information for the newsletter feature."""
+
+    current_count: int = Field(description="Current number of active newsletters/tasks.")
+    max_allowed: int = Field(description="Maximum number of active newsletters/tasks allowed for the user's plan.")
+
+
 class UserInfoResponse(BaseModel):
     """Response model for user information endpoint."""
 
     subscription_info: dict[str, Any] = Field(description="Raw subscription info from Dodo Payments API")
     plan_name: str = Field(description="User's plan name (PRO, BETA, etc.)")
-    usage_info: UsageInfo = Field(description="User's current usage information")
+    usage_info: UsageInfo = Field(description="User's current usage information for email processing.")
+    newsletter_usage: NewsletterUsageInfo = Field(description="User's current usage for the newsletter feature.")
+
+
+class ScheduleType(str, Enum):
+    IMMEDIATE = "IMMEDIATE"
+    SPECIFIC_DATES = "SPECIFIC_DATES"
+    RECURRING_WEEKLY = "RECURRING_WEEKLY"
+
+
+class RecurringWeekly(BaseModel):
+    days: list[str] = Field(..., description="List of weekdays, e.g., ['monday', 'wednesday', 'friday']")
```

Comment:
the days input can just be integers instead of strings like monday tuesday, the client can take care of converting monday tuesday to integers, that simplifies the backend logic